extendable_fastapi/models.py

```diff
@@ -0,0 +1,20 @@
+from extendable_pydantic import ExtendableBaseModel
+
+
+class StrictExtendableBaseModel(
+    ExtendableBaseModel,
+    revalidate_instances="always",
+    validate_assignment=True,
+    extra="forbid",
+    strict=True,
+):
+    """
+    Add Config parameters to ExtendableBaseModel:
```

Comment:
Before explaining the how, it's worth explaining the what and the why ;-)

```suggestion

    By default, Pydantic does not revalidate instances during validation, nor
    when the data is changed. Validation only occurs when the model is created.
    This is not suitable for a REST API, where the data is changed after the
    model is created or the model is created from a partial set of data and 
    then updated with more data. This class enforce strict validation by
    forcing the revalidation of instances when the method `model_validate` is
    called and by ensuring that the values assignment is validated.
    
    Add Config parameters to ExtendableBaseModel:
```

Reply:
Added, thanks